src/node_file.cc

```diff
@@ -992,6 +993,23 @@ void Close(const FunctionCallbackInfo<Value>& args) {
   }
 }
 
+static void FastClose(Local<Object> recv,
+                      const int32_t fd,
+                      // NOLINTNEXTLINE(runtime/references) This is V8 api.
+                      v8::FastApiCallbackOptions& options) {
+  Environment* env = Environment::GetCurrent(recv->GetCreationContextChecked());
+
+  env->RemoveUnmanagedFd(fd);
+
+  uv_fs_t req;
+  FS_SYNC_TRACE_BEGIN(close);
```

Comment:
Should we document if this function is deoptimized, this will generate two traces for the same operation?

Reply:
I'm not really quite sure, what to do in this case.